addons/website/static/src/components/dialog/edit_menu.js

```diff
@@ -109,7 +109,7 @@ export class EditMenuDialog extends Component {
 
         this.menuEditor = useRef('menu-editor');
 
-        this.state = useState({ rootMenu: {} });
+        this.state = useState({ rootMenu: {}, map: new Map() });
```

Comment:
I see map isn't used in the view template, so i don't think maintaining the state of the map is a good way. the  `this.state.rootMenu` is responsible for the data being rendered. May be that state is not being change which cause this issue. I am not sure but we should research on that way... Thank : )

Reply:
Thanks...I'll explore some alternative solutions to this.